### PR TITLE
Side navigation

### PR DESCRIPTION
**Pull request by ABucin**
# Description

Added some missing exports for the side navigation component.

### CODE REVIEW

**Review by paul-request — DISMISSED** [review]

**Comment by elenagarrone** [comment]
@AlexandruHoisan if you have time would you mind updating the colour variables as part of this PR? Not sure if you have seen it but I mentioned in the slack channel yesterday :)

This was my comment:
```
[...]. Just two small things to make note of/fix:
the colour variables (e.g. --color-midnight-blue) in side-nav-bar-item.component.scss should be a bit made a bit more generic so that we can easily apply branding.
the --color-platinum for the borders could be changed to use --border-color  instead
```

**Review by paul-request — DISMISSED** [review]

File: projects/canopy/src/lib/side-nav/side-nav-bar-item/side-nav-bar-item.component.scss [hunk 1]
```diff
@@ -3,10 +3,10 @@
 .lg-side-nav-bar-item {
   &:hover {
     .lg-side-nav-bar-item__inner {
-      color: var(--color-midnight-blue);
+      color: var(--side-nav-bar-hover);
```
Comment (elenagarrone) [inline projects/canopy/src/lib/side-nav/side-nav-bar-item/side-nav-bar-item.component.scss:6]: we usually append the property name at the end `--side-nav-bar-hover-color`

File: projects/canopy/src/lib/side-nav/side-nav-bar-item/side-nav-bar-item.component.scss [hunk 2]
```diff
@@ -54,8 +54,9 @@
 
 .lg-side-nav-bar-item--selected {
   .lg-side-nav-bar-item__inner {
-    color: var(--color-super-blue);
-    border-bottom: var(--side-nav-bar-item-border-width-md) solid var(--color-super-blue);
+    color: var(--side-nav-bar-active);
```
Comment (elenagarrone) [inline projects/canopy/src/lib/side-nav/side-nav-bar-item/side-nav-bar-item.component.scss:57]: `--side-nav-bar-active-color`

File: projects/canopy/src/lib/side-nav/side-nav-bar-item/side-nav-bar-item.component.scss [hunk 3]
```diff
@@ -54,8 +54,9 @@
...
+    border-bottom: var(--side-nav-bar-item-border-width-md) solid
+      var(--side-nav-bar-active);
```
Comment (elenagarrone) [inline projects/canopy/src/lib/side-nav/side-nav-bar-item/side-nav-bar-item.component.scss:59]: In this case I think it's fine to use the same variable as above `--side-nav-bar-active-color`

File: projects/canopy/src/lib/side-nav/side-nav-bar-item/side-nav-bar-item.component.scss [hunk 4]
```diff
@@ -3,10 +3,10 @@
 .lg-side-nav-bar-item {
   &:hover {
     .lg-side-nav-bar-item__inner {
-      color: var(--color-midnight-blue);
+      color: var(--side-nav-bar-hover);
       cursor: pointer;
       border-bottom: var(--side-nav-bar-item-border-width-lg) solid
-        var(--color-midnight-blue);
+        var(--side-nav-bar-hover);
```
Comment (elenagarrone) [inline projects/canopy/src/lib/side-nav/side-nav-bar-item/side-nav-bar-item.component.scss:9]: Just use `var(--side-nav-bar-hover-color)` here too

**Review by elenagarrone — APPROVED** [review]
Thanks a lot for updating the variables!

**Review by andy-polhill — APPROVED** [review]

**Pull request merged**